### PR TITLE
Add lang according to bo locale for datetimepicker

### PR DESCRIPTION
**Pull request by fkroockmann**
Currently the datetimepicker is all the time in english

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
such amazing ! :+1:

File: src/tb/component/formbuilder/form/element/views/form.element.view.datetimepicker.js [hunk 1]
```diff
@@ -80,7 +81,8 @@ define(
                     parentID: '#bb5-dialog-container',
                     onGenerate: function () {
                         jQuery(this).css('top', element.position().top + (element.innerHeight() * 2.5));
-                    }
+                    },
+                    lang: Translator.getLocale().substring(0, 2)
```
Comment (crouillon) [inline src/tb/component/formbuilder/form/element/views/form.element.view.datetimepicker.js]: What happens if lang isn't recognize?

Comment (fkroockmann) [inline src/tb/component/formbuilder/form/element/views/form.element.view.datetimepicker.js]: The datetimepicker doesn't work but this datetimepicker accept a lot of lang

Comment (mickaelandrieu) [inline src/tb/component/formbuilder/form/element/views/form.element.view.datetimepicker.js]: at least, we only have french, english, spanish, russian, german and chineses (chinese) languages available in toolbar: I'm pretty sur the datetime picker can handle it.

Comment (fkroockmann) [inline src/tb/component/formbuilder/form/element/views/form.element.view.datetimepicker.js]: I will change my PR, i found available lang in datetimepicker

**Comment by fkroockmann** [comment]
ping @crouillon @mickaelandrieu , if the lang is not found, the default lang is 'en'

**Comment by crouillon** [comment]
thx @fkroockmann :+1:

**Pull request merged**